package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "volto-slate",
-  "version": "5.3.4",
+  "version": "5.3.5",
```

Comment:
@sneridagh @tiberiuichim Is this ok as a patch, or it should be a minor release?

Reply:
(but allowing to get the right upstream changes, and other add-ons config patches)